addons/knowledge/static/src/js/widgets/knowledge_dialogs.js

```diff
@@ -0,0 +1,186 @@
+odoo.define('knowledge.dialogs', function (require) {
```

Comment:
File is called "knowledge_dialogs" but it only contains the move article dialog?